### PR TITLE
CDSID-557 | authorisation holder page

### PR DESCRIPTION
**Pull request by gorkemunsal**

### CODE REVIEW

File: app/views/authorisation_holder.scala.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+@main_template(title = messages("authorisationHolder.title"), bodyClasses = None) {
+
+@helpers.form(action = AuthorisationHoldersController.onSubmit()) {
+
+    @components.back_link("")
+
+    @components.error_summary(form.errors)
+
+    <h1 class="heading-medium">@messages("authorisationHolder.heading")</h1>
```
Comment (russellthorn) [inline app/views/authorisation_holder.scala.html]: Minor thing, but the norm is class for h1's is `heading-large` but for what we're building here it's kind of nit picking.

File: app/views/authorisation_holder.scala.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    @if(authorisationHolders.nonEmpty) {
+        @if(authorisationHolders.size > 1){
+            <p>@authorisationHolders.size @messages("authorisationHolder.table.multiple.heading")</p>
+        } else {
+            <p>@authorisationHolders.size @messages("authorisationHolder.table.heading")</p>
+        }
+
+        <table>
+            <thead>
+                <th scope="col">@messages("authorisationHolder.id")</th>
```
Comment (russellthorn) [inline app/views/authorisation_holder.scala.html]: These `th`'s need wrapping with a `<tr>`

File: app/views/authorisation_holder.scala.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                <td>@item.categoryCode</td>
+            </tr>
+            }
+            </tbody>
+        </table>
+        } else {
+        <p>@messages("authorisationHolder.table.empty")</p>
+        }
+
+    <div class="grid-row">
```
Comment (russellthorn) [inline app/views/authorisation_holder.scala.html]: I've not check this on localhost, but you shouldn't need these parent wrappers for the fieldset `<div class="grid-row">` and `<div class="column-two-thirds">` these simply set the width of content to 2/3's when `content__body` is set to 100% width with a modifier class, which isn't the case for this page as far as I can see.

File: app/views/components/error_summary.scala.html [hunk 4]
```diff
@@ -19,11 +19,9 @@
     <div class="error-summary error-summary--show" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
 
         <h2 class="h2-heading" id="error-summary-heading">
```
Comment (russellthorn) [inline app/views/components/error_summary.scala.html]: class `h2-heading` didn't know this was a thing? `heading-medium` is usually used. For all I know they do the same thing? If it looks ok to you you could leave it, but wont take long to update for consistency.

File: app/views/authorisation_holder.scala.html (hunk 1 above)
Comment (kurtlogan) [reply app/views/authorisation_holder.scala.html]: Done

File: app/views/authorisation_holder.scala.html (hunk 2 above)
Comment (kurtlogan) [reply app/views/authorisation_holder.scala.html]: done

**Pull request merged**